core/src/main/java/org/stellar/anchor/config/SecretConfig.java

```diff
@@ -1,20 +1,27 @@
 package org.stellar.anchor.config;
 
+import javax.crypto.SecretKey;
+
 public interface SecretConfig {
-  String getSep6MoreInfoUrlJwtSecret();
+  SecretKey getSep6MoreInfoUrlJwtSecret();
 
-  String getSep10JwtSecretKey();
+  @Deprecated // TODO: ANCHOR-667
+  SecretKey getSep10JwtSecretKey();
 
   String getSep10SigningSeed();
 
-  String getSep24InteractiveUrlJwtSecret();
+  SecretKey getSep24InteractiveUrlJwtSecret();
```

Comment:
Question: why do some of these `SecretKey`s end with `key` but others do not?

Reply:
I will remove all added fields and rename old ones back

Reply:
I thought we decided to remove the `SecretKey`s?

Reply:
Sorry I forgot to push 🤦🏼